lib/common_test/src/test_server_ctrl.erl

```diff
@@ -1124,6 +1124,7 @@ init_tester(Mod, Func, Args, Dir, Name, {_,_,MinLev}=Levels,
 				 {reject_io_reqs,RejectIoReqs}]),
     group_leader(test_server_io:get_gl(true), self()),
     {TimeMy,Result} = ts_tc(Mod, Func, Args),
+	print(major, "=suite_time    ~w", [TimeMy]),
```

Comment:
While at it, please improve variable name. `TimeMy` is an antipattern I'm afraid.

Reply:
Changed to ElapsedTime